api/core/model_runtime/model_providers/tongyi/llm/qwen-max-1201.yaml

```diff
@@ -8,54 +8,65 @@ model_properties:
 parameter_rules:
   - name: temperature
     use_template: temperature
-    default: 1.0
+    type: float
+    default: 0.85
```

Comment:
Btw, the default temperature seems too high. In our practice on Qwen, it will make LLM's answer unrelated to the recalled knowledge. Would prefer setting relatively lower in 0.1 for more stable answer.